lib/tranzito_utils/helpers/helpers.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+module TranzitoUtils
+  module Helpers
+    include TranzitoUtils::SortableHelper
+    include TranzitoUtils::GraphingHelper
+
+    def controller_namespace
+      @controller_namespace ||= self.class.module_parent.name != "Object" ? self.class.module_parent.name.downcase : nil
+    end
+
+    def in_admin?
+      controller_namespace == "admin"
+    end
+
+    def page_title
+      return @page_title if defined?(@page_title)
+      prefix = (in_admin? ? "🧰" : Rails.application.engine_name.split("_").first.capitalize)
```

Comment:
`application_display_name` should be something you can set in the `tranzito_utils` config.

Also, the current method returns "Bike" for BikeHub.

Reply:
Yeah, I think the config method is fine will do it.